Source/WebKit/UIProcess/WebPageProxy.cpp

```diff
@@ -5546,15 +5576,13 @@ void WebPageProxy::updateRemoteFrameSize(WebCore::FrameIdentifier frameID, WebCo
 {
     if (!drawingArea())
         return;
-    auto* frame = WebFrameProxy::webFrame(frameID);
+    RefPtr frame = WebFrameProxy::webFrame(frameID);
     if (!frame) {
         ASSERT_NOT_REACHED();
         return;
     }
-    auto remotePageProxy = internals().domainToRemotePageProxyMap.get(RegistrableDomain(frame->url()));
-    if (!remotePageProxy)
-        return;
-    remotePageProxy->send(Messages::WebPage::UpdateFrameSize(frameID, size));
+    if (auto remotePageProxy = internals().domainToRemotePageProxyMap.get(RegistrableDomain(frame->url())))
```

Comment:
Why not use RefPtr here.

Reply:
Oh, I can do that too. The `auto` here is ` WeakPtr` but I can call `.get()` and store in a RefPtr right away.

Reply:
I think that would be more idiomatic.